src/Library/NoticeCounts.php

```diff
@@ -0,0 +1,208 @@
+<?php //phpcs:ignore WordPress.Files.FileName
+/**
+ * BoldGrid Library Notice Counts
+ *
+ * @package Boldgrid\Library
+ * @subpackage \Library
+ *
+ * @version 1.0.0
+ * @author BoldGrid <wpb@boldgrid.com>
+ */
+
+namespace Boldgrid\Library\Library;
+
+/**
+ * BoldGrid Library Notice Counts Class.
+ *
+ * This class is responsible for adding any unread notice count numbers
+ * that are displayed in the WordPress dashboard.
+ *
+ * @since 1.0.0
+ */
+class NoticeCounts {
+	/**
+	 * Option Name
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var string
+	 */
+
+	public $option_name = 'boldgrid-plugin-notice-counts';
+
+	/**
+	 * Get Core Configs
+	 * gets configs array from admin core
+	 * exists;
+	 *
+	 * @since 1.0
+	 * @var array
+	 */
+	private $configs;
+
+	/**
+	 * Get Unread Count.
+	 *
+	 * @param string $id of the notice count type to return.
+	 * @since 1.0
+	 * @return string.
+	 */
+	public static function get_unread_count( $id ) {
+		$option = get_option( 'boldgrid-plugin-notice-counts' );
+		if ( $option && isset( $option[ $id ] ) ) {
+			$unread_count = 0;
+			$notices      = $option[ $id ];
+			foreach ( $notices as $notice ) {
+				if ( true === $notice ) {
+					$unread_count++;
+				}
+			}
+			return self::count_markup( $unread_count );
+		}
+	}
+
+	/**
+	 * Get Total Unread.
+	 *
+	 * Determines the total number of unread notices
+	 *
+	 * @since 1.0
+	 * @return string.
+	 */
+	public function get_total_unread() {
+		$option       = get_option( 'boldgrid-plugin-notice-counts' );
+		$total_unread = 0;
+		if ( $option ) {
+			$config_notices = $this->configs[ $this->option_name ];
+			foreach ( array_keys( $config_notices ) as $notice_type ) {
+				if ( isset( $option[ $notice_type ] ) ) {
+					foreach ( $option[ $notice_type ] as $is_unread ) {
+						if ( $is_unread ) {
+							$total_unread++;
+						}
+					}
+				}
+			}
+		}
+		return $this->count_markup( $total_unread );
+	}
+
+	/**
+	 * Unread Count Markup.
+	 * generates HTML Markup for unread notice counts
+	 *
+	 * @param int $count number to use for markup.
+	 * @return string the marked up unread count string.
+	 * @since 1.0
+	 */
+	public static function count_markup( $count ) {
```

Comment:
`count_markup()` sounds like it's going to count something up for us. What do you think about `get_count_markup()`?